Source/IntegrationTests/LitTests.cs

```diff
@@ -151,6 +151,7 @@ public class LitTests {
     [FileTheory]
     [FileData(Includes = new[] { "**/*.dfy", "**/*.transcript" },
               Excludes = new[] { "**/Inputs/**/*", "**/Output/**/*",
+                "**/libraries/**",
```

Comment:
This shouldn't be necessary now that https://github.com/dafny-lang/libraries/pull/97 is merged if we update the submodule pointer, right?

Reply:
Well, it seemed odd that the CI tests should run lit on the library when there is a library nightly that runs anyway. So it seems they ought to be excluded here.

Reply:
The original intention of adding the submodule was to ensure Dafny changes didn't break that code, so it's not the same as relying on that code testing itself as it evolves. I'm not 100% sure that makes sense and we could consider dropping it (i.e. we're moving towards testing against nightly Dafny releases to catch such breakages instead), but not as part of the scope of this PR.

Reply:
OK - I'll push a commit that omits that exclusion.